assets/script/object.js

```diff
@@ -0,0 +1,235 @@
+    description1:
+      "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s.",
+    technologies: {
+      tech1: "html",
+      tech4: "Ruby on rials",
+      tech2: "css",
+      tech3: "javaScript",
+    },
+    details: "See project",
+    imageMobile: "./assets/img/cards/card3.png",
+    imageDesktop: "./assets/img/cards/card3.png",
+    live: "https://wilfriedm01.github.io/",
+    source: "https://github.com/WilfriedM01/portfolio-project.git",
+  },
+  {
+    id: "card4",
+    title: "Multi-Post<br>Stories",
+    title1: "Uber Navigation",
+    company: "CANOPY",
+    company1: "Uber",
+    icon: "./assets/icons/modal_git_icon.svg",
+    icon1: "./assets/icons/modal_export_icon.svg",
+    specialization: "Back End Dev",
+    specialization1: "Lead Developer",
+    year: "2015",
+    year1: "2018",
+    description:
+      'A daily selection of privately <br class="must-ignor">personalized reads;no accounts or <br class="must-ignor">sign-ups required.',
+    description1:
+      "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book.",
+    technologies: {
+      tech1: "html",
+      tech4: "Ruby on rials",
+      tech2: "css",
+      tech3: "javaScript",
+    },
+    details: "See project",
+    imageMobile: "./assets/img/cards/card4.png",
+    imageDesktop: "./assets/img/cards/card4.png",
+    live: "https://wilfriedm01.github.io/",
+    source: "https://github.com/WilfriedM01/portfolio-project.git",
+  },
+];
+
+// Work section data structure
+const Worksection = document.querySelector(".work-cards");
+projects.forEach((project) => {
+  const cardDiv = document.createElement("div");
+  cardDiv.innerHTML = `
+  <a href="#" class="article-link opacity"
+        ><section class="articles">
+          <article class="article-item ${project.id}">
+            <p class="sub-item">
+              <img
+                src="${project.imageDesktop}"
+                alt=""
+                class="article-img"
+              />
+            </p>
+            <div class="article-content">
+              <h2 class="article-title">${project.title}</h2>
+              <h3>${project.company} <span>&bull; ${project.specialization} &bull; ${project.year}</span></h3>
+              <p class="article-text">
+              ${project.description1}
+              </p>
+              <ul class="article-tags">
+                <li class="article-tag ok">${project.technologies.tech1}</li>
+                <li class="article-tag">${project.technologies.tech2}</li>
+                <li class="article-tag">${project.technologies.tech3}</li>
+              </ul>

```

Comment:
- [ ] [Optional] 👏👏 Great job on implementing the project description functionality! 🌟 Your projects object contains four technologies for each project, but currently, you are only showing three. To enhance the flexibility of your project description, consider adding functionality to dynamically display all the technologies for each project. 

By making this change, your project description will become more robust and user-friendly, allowing you to showcase all the necessary information for each project. Keep up the good work, and happy coding! 🚀💻